basics/lists-and-tuples.md

```diff
@@ -128,6 +131,26 @@ Note that `remove` removes only the first match it finds.
 >>>
 ```
 
+The method `pop` also works for delete elements of the list.
+
+```python
+>>> names = ['theelous3', 'go|dfish', 'theelous3']
+>>> names.pop(1)
+>>> names    # the second item was removed
+'go|dfish'
+>>> names
+['theelous3', 'theelous3']
+
+>>> names = ['theelous3', 'go|dfish', 'theelous3']
+>>> names.pop()
+theelous3'
+>>> names
+['theelous3', 'go|dfish']
+
+>>>
+```
+
+
```

Comment:
Can you add an explanation about how `pop` is different from `remove`? Also, I think `names.pop(1)` is a thing that isn't needed very often in real-world programming (like `insert`), but popping the last thing with `names.pop()` is more useful.